### ISSUE TITLE
Underwater rock hides a Land area.

### ISSUE THREAD

**Issue by Christian-Shom** [S-101-Portrayal-subWG/Working-Documents#99]
From ENCWG "Display Sub WG"
**Underwater rock** symbol covers (hides) point **Land area** (which is a more dangerous feature). 
![image](https://user-images.githubusercontent.com/71139640/181256130-c0cae062-2bf4-4311-a4c8-a83488150b48.png)
This has not been ckecked in S-101. If confirmed, could this be improved playing with the Display Priority: a feature should never mask a more dangerous one?
In this specific case, there may be a compilation issue: is it necessary to show the Underwater rock so close to the island? 
But there may be other situations that need to be improved. Probably needs further investigations.

**Comment by alvarosanuy** [comment]
**Portrayal subWG meeting - 12th January 2023**

1. It seems to be a problem with the priority allocated to **LandArea** features of type Point.

2. The subWG decided to experiment with altering the DP on a test data set. It is important to confirm that unwanted results on other features are not accidentally triggered by this change. 

3. **Norway** will provide the S-101 ENC test data set and will perform the assessment and report findings at the next subWG meeting. **NIWC** is to assist Norway by providing a PC version that has a higher priority allocated to **LandArea** point features.

**Comment by TomRichardson6** [comment]
Test Dataset 13 Edtn 5 has been updated to try and reflect this scenario

https://github.com/iho-ohi/S-101-Test-Datasets/tree/main/dev/cells/101AA00DS0013

**Comment by DavidGrant-NIWC** [comment]
### TL;DR
To ensure `LandArea` is always on top of any `UnderwaterAwashRock` (and other potential isolated dangers) would require a drawing priority greater than `24` (`8` in S-52), or changing the priority of many features. Unfortunately, text and the danger highlight also share priority `24`; moving `LandArea` to `25` would obscure text, and cause the feature to obscure the danger highlight.

---
### Details
* Appearance of the `UnderwaterAwashRock` depends on the values selected for the `safety contour` and the `safety depth`.
* You may wish to encode other representative features in the area so their stacking order can be evaluated.

|`Selected Safety Contour`|`Selected Safety Depth`|`UnderwaterAwashRock`|`LandArea`|Result|
|:---:|:---:|:---:|:---:|:---:|
|30|30|Priority=`24`|Priority=`12`|![image](https://user-images.githubusercontent.com/62906211/229532594-c80536c0-db66-46ed-a05c-e0578e252056.png)|
|0.5|30|Priority=`12`|Priority=`12`|![image](https://user-images.githubusercontent.com/62906211/229532809-3bfa1682-e808-4efc-92b4-a5bfb3155dbc.png)|
|0.5|0.5|Priority=`12`|Priority=`12`|![image](https://user-images.githubusercontent.com/62906211/229552543-6b24e843-86d2-4468-95c9-fa9e5d2fc6da.png)|

Note: Even with the safety contour set to `0` the safety checks will detect the `LandArea` (provided the route intersects the point):
![image](https://user-images.githubusercontent.com/62906211/229535542-7893759b-77df-4109-b6a2-dfae09d3f6ac.png)
![image](https://user-images.githubusercontent.com/62906211/229535635-baa97c1c-74b4-4b3d-a904-369bdb08462d.png)

**Comment by TDYCARHugh** [comment]
This will also be an issue in S-52.  I don't think this can be solved by adjusting priorities.  With a small land area surface the coastline draws at a higher priority so it can still be seen. So as stated above the issue is with a point land area. The symbol covering other data is the DANGER01 symbol which is filling the oval with a solid DEPVS colour and masking anything underneath.  Keep in mind that DANGER01 is only used when the surrounding area is shallow water. Maybe we could try using a transparent fill for DANGER01.  With a 50% transparency the land area point can still be seen.

The first image below is using the current DANGER01 symbol.  The land area surface (upper left) coastline can still be seen. The Land Area point (upper right) is hidden.
In the second image DANGER01 is using a 50% transparency for the fill. (style="fill-opacity:0.5;")

<img width="50" alt="DANGER01_Solid" src="https://user-images.githubusercontent.com/50683133/229802577-362d4ef1-d19a-4127-bb2b-e095192ec14b.png">
<img width="50" alt="DANGER01_50" src="https://user-images.githubusercontent.com/50683133/229802606-2567ed7b-123d-4815-a418-da602263d7dd.png">

**Comment by DavidGrant-NIWC** [comment]
> With a 50% transparency the land area point can still be seen.

That could work, but bear in mind that as additional danger objects are stacked the fill would become progressively more opaque.

**Comment by alvarosanuy** [comment]
> That could work, but bear in mind that as additional danger objects are stacked the fill would become progressively more opaque.

I hope SCAMIN kicks in and symbology is cleared up as the ENC is zoomed out .... I'm of the opinion that, if cartographers attribute SCAMIN (as a chart generalisation tool) **based on what they see at each comp. scale**, we wouldn't be talking about these overlaps being a problem .....

Having said this, I'm happy to pursue the recommendation presented by Hugh (50% transparency).

**Comment by alvarosanuy** [comment]
**Decisions made at Portrayal subWG meeting on 11/5/23**

- It was agreed that the problem is more on the compilation and encoding side that on portrayal.

- [x] **Alvaro** - Refer this issue to the NCWG so it can be discussed and improved compilation and ENC encoding (use of SCAMIN) guidance be added to S-4. New guidance must highlight the different results obtained when compiling thinking on paper charts and then using the same data in an ENC (with <> comp. scale and symbol sizes). It's recommended that HO's compile thinking on the look and feel of the area in an ENC at comp. scale (or M_CSCL scale). SCAMIN is a key tool to avoid scenarios like the one reported in this issue. In this case, the UWRTOC needs to disappear from the screen when it gets too close and clashes (and obscures) land. If this happens at comp. scale then the UWTROC shouldn't be captured at all (just make sure it's covered by the right DEPARE).

**Comment by alvarosanuy** [comment]
Paper submitted to NCWG9 (Nov 2023).
Will leave this issue open until we get feedback from NCWG Chair.

**Comment by MatthewCraggs** [comment]
### Below are the comments from NCWG9.
**_Underwater rocks, hides land area point features, meaning the more dangerous feature is being overridden 
by a less dangerous feature, how to proceed?_**

Important to note that no unanticipated unwanted changes must be allowed to occur as a result.
Changing the display priority on test dataset has been considered and tested, additional new complications 
were created, adjusting the priority is not the solution. 
Options, SCAMIN, transparency, extend the land over the rock, capture a danger line around the island, 
capture the island in the true position and offset the rock, 

**_Outcome:_** NCWG recommends improved encoding guidance rather than offsetting features from their true 
position is the better solution.

**Comment by alvarosanuy** [comment]
NCWG decided not to amend symbology or portrayal rules.
